net/ksmbd-tools/files/ksmbd.init

```diff
@@ -40,6 +62,7 @@ smb_header()
 	    -e "s#|INTERFACES|#$interfaces#g" \
 	    /etc/ksmbd/ksmbd.conf.template > /var/etc/ksmbd/ksmbd.conf
 		
+	config_get_ram >> /var/etc/ksmbd/ksmbd.conf
```

Comment:
This can get placed in the {} block below.

Reply:
This is also done, I've put it before `######### Dynamic written config options #########`